target/linux/ath79/dts/qca9563_tplink_cpe710-v1.dts

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca956x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "TP-Link CPE710 v1";
+	compatible = "tplink,cpe710-v1", "qca,qca9563";
+
+	aliases {
+		label-mac-device = &eth0;
+		led-boot = &led_lan;
+		led-failsafe = &led_lan;
+		led-upgrade = &led_lan;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_lan: lan {
+			label = "blue:lan";
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan5g {
+			label = "blue:wlan5g";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
```

Comment:
Missing label.

Reply:
@adschm I have added the reset label back as requested. I did have it but @mpratt14 asked me to remove it.

Reply:
@adschm maybe this is something you would rather handle as a treewide commit to edit DTS files, like many other you have done...

according to Linux docs, the label property is deprecated and when it is missing, the node name is taken as the label

we still need labels for LEDs...unless we come up with a common node name scheme instead
but I know for sure with gpio-keys it makes no difference

Reply:
I'd be open to dropping label on keys, since it's much more redundant here and in contrast to LEDs, it would be actually possible to use the node name. But we first should make sure that this actually applies to keys and applies for kernel 5.4.

Reply:
@adschm I looked deeper into the label thing and part of what I said was wrong...

for gpio-leds:
label is considered deprecated, but we currently need it
see:
https://www.kernel.org/doc/Documentation/devicetree/bindings/leds/common.yaml

for gpio-keys:
label is not deprecated, but is used only as a "description" and is optional, and we do not need it
when label is missing, the string "gpio_keys" is the default value
however, to use the node name as the label would be a very easy patch
(converting 1 line to an if / else statement)
see lines 484 and 731 or search "button->desc":
https://elixir.bootlin.com/linux/v5.4.119/source/drivers/input/keyboard/gpio_keys.c

Reply:
@mpratt14 I'd support such a change, but only if it's in kernel (which we could backport then).
If you intend to submit, please put me in Cc. I'd personally frame this as achieving consistency with the behavior for LEDs, but I have not much experience myself in what the kernel people would like to hear.